### PR TITLE
Add unit tests for decimal not-equal functions

### PR DESCRIPTION
**Pull request by karteekmurthys**
The neq functions are registered as part of this PR https://github.com/facebookincubator/velox/pull/4638.
This change adds some unit tests for neq decimal function.

### CODE REVIEW

File: velox/functions/prestosql/tests/ComparisonsTest.cpp [hunk 1]
```diff
@@ -132,11 +132,12 @@ TEST_F(ComparisonsTest, betweenDecimal) {
       "between(DECIMAL(20,2), DECIMAL(3,2), DECIMAL(3,2)).");
 }
 
-TEST_F(ComparisonsTest, eqDecimal) {
+TEST_F(ComparisonsTest, eqNeqDecimal) {
   auto runAndCompare = [&](std::vector<VectorPtr>& inputs,
-                           VectorPtr expectedResult) {
-    auto actual =
-        evaluate<SimpleVector<bool>>("c0 == c1", makeRowVector(inputs));
+                           VectorPtr expectedResult,
```
Comment (laithsakka) [inline velox/functions/prestosql/tests/ComparisonsTest.cpp:137]: nit: const VectorPtr& 
nit: const std::vector<VectorPtr>& inputs

**Review by laithsakka — APPROVED** [review]

**Comment by laithsakka** [comment]
Looks good just a small nit

File: velox/functions/prestosql/tests/ComparisonsTest.cpp (hunk 1 above)
Comment (karteekmurthys) [reply velox/functions/prestosql/tests/ComparisonsTest.cpp]: Done.

**Comment by majetideepak** [comment]
@karteekmurthys can we hold on to merging this PR until this  https://github.com/facebookincubator/velox/pull/4434 lands?
Thanks.

**Review by majetideepak — CHANGES_REQUESTED** [review]
Would like to block this PR until https://github.com/facebookincubator/velox/pull/4434 lands.

**Comment by majetideepak** [comment]
The reason to block is that https://github.com/facebookincubator/velox/pull/4434 implements decimal type as a logical type and it impacts all decimal code changes. We want to minimize the diff.

**Review by majetideepak — APPROVED** [review]
@karteekmurthys CI build is failing.

**Pull request closed without merging**